server/backend/testspg/jwt_utils.py

```diff
@@ -29,29 +34,35 @@ def create_jwt_claims():
         "exp": time.time() + 60000,
         "iat": time.time(),
         "jti": "6ab8647c-0679-4d25-a71a-2400966fea9a",
-        "username": COGNITO_USERNAME
+        "username": COGNITO_USERNAME,
     }
 
 
 def create_jwt_token(
     test_rsa_key,
     roles=["FAM_ADMIN"],
+    username=COGNITO_USERNAME,
     claims=create_jwt_claims(),
-    test_algorithm='RS256',
+    test_algorithm="RS256",
     test_headers={"kid": "12345"},
 ):
     if roles is not None:
         claims["cognito:groups"] = roles
-    return jws.sign(claims, test_rsa_key, algorithm=test_algorithm,
-                    headers=test_headers)
+    claims["username"] = username
```

Comment:
add an option to pass the cognito user name of a business bceid user, so we can create a token for a business bceid user